src/System.Management.Automation/FormatAndOutput/common/ComplexWriter.cs

```diff
@@ -133,6 +133,12 @@ private void WriteToScreen()
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Native streams behavior with PowerShell.Error stream.preserves error stream as is with Out-String
```powershell
Expected strings to be the same, but they were different.
Expected length: 33
Actual length:   29
Strings differ at index 4.
Expected: 'foo\n\nbar\n\nbazmiddlefoo\n\nbar\n\nbaz\n'
But was:  'foo\nbar\nbazmiddlefoo\nbar\nbaz\n'
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeStreams.Tests.ps1: line 52
52:             ($out | Out-String).Replace("`r", '') | Should -BeExactly "foo`n`nbar`n`nbazmiddlefoo`n`nbar`n`nbaz`n"
```